dev-lang/micropython/micropython-1.25.0.ebuild

```
@@ -0,0 +1,79 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Python implementation for microcontrollers"
+HOMEPAGE="https://micropython.org https://github.com/micropython/micropython"
+SRC_URI="https://micropython.org/resources/source/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	dev-libs/libffi:=
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	default
+	cd "${S}/ports/unix" || die
+
+	# 1) don't die on compiler warning
+	# 2) remove /usr/local prefix references in favour of /usr
+	# 3) enforce our CFLAGS (Only change the first `CFLAGS +=`)
+	# 4) enforce our LDFLAGS (Only change the first `LDFLAGS +=`)
+	sed -e 's#-Werror##g;' \
+		-e 's#\/usr\/local#\/usr#g;' \
```

```suggestion
		-e "s#/usr/local#${EPREFIX}/usr#g;" \
```

I don't think there's any need to escape forward slashes if you're using other delimiter.

---

I think you're right on this. I left that line from the original dev assuming there was some wisdom they knew that I didn't